scargo/docker_base/esp32.Dockerfile

```diff
@@ -0,0 +1,114 @@
+###
+# @copyright Copyright (C) 2022 Spyrosoft Solutions. All rights reserved.
+# THIS FILE WAS GENERATED AUTOMATICALLY. DO NOT CHANGE IT.
+###
+
+ARG DOCKER_IMAGE_ROOT=ubuntu:20.04
+FROM ${DOCKER_IMAGE_ROOT} as base
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt update --fix-missing && \
```

Comment:
In each docker file you multipying base layer and the <project>_dev dev layer. It will be hard for maintenance as need same changes in three separate files